### DISCUSSION TITLE
Custom sorting of included fields

### DISCUSSION THREAD

**Discussion by harrislop** [marcusolsson/obsidian-projects#959] [General]
<img width="214" alt="제목 없음" src="https://github.com/user-attachments/assets/6f736347-e76c-4b32-b36e-c622516fa247">

In Notion, when displaying properties in a gallery or board, users can set the order by dragging and dropping. Is there a way to set the order manually, even if it’s not through drag-and-drop?

**Comment by Acylation** [comment]
This can be achieved by drag n drop ordering columns in table view.

**Reply by harrislop** [reply]
It seems that even when performing actions in table view, the order in the gallery remains fixed in alphabetical order.
What am I doing wrong?

**Reply by Acylation** [reply]
Oh you're correct, seems like reordering only works for table view. Will do some investigation. Do you mind me transform this post into a feature request?

**Reply by harrislop** [reply]
Yes! Thank you
